src/presentation/pages/Login.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Navbar from '../components/Navbar'
+import { Button, Divider, Input } from '@nextui-org/react'
+import { Link } from 'react-router-dom'
+import LinkIcon from '../assets/LinkIcon'
+
+export default function Login() {
+  return (
+    <>
+    {/* FIX: acertar largura dos componentes na página e suas responsividades */}
+    {/* FEATURE: conectar ao banco e fazer checagens necessárias p/ funcionamento do botão de login*/}
+    {/* FIX: remover navbar? */}
+      <Navbar />
+        <main className="xs:mx-5 md:mx-20 lg:mx-72">
```

Comment:
Substituir aqui pela nova estilização container

```suggestion
        <main className="container">
```